projects/storefrontlib/cms-components/anonymous-consent-management/banner/anonymous-consent-management-banner.component.html

```diff
@@ -23,6 +23,29 @@
           </button>
         </div>
       </div>
+      <div class="row" *cxFeature="'a11yHideConsentButtonWhenBannerVisible'">
+        <div class="col-xl-7 col-lg-6 col-xs-12">
+          <div class="cx-banner-title">
+            {{ 'anonymousConsents.banner.title' | cxTranslate }}
+          </div>
+          <div class="cx-banner-description">
+            {{ 'anonymousConsents.banner.description' | cxTranslate }}
+          </div>
+        </div>
+
+        <div class="col-xl-5 col-lg-6 col-xs-12 cx-banner-buttons">
+          <button
+            *cxFeature="'a11yHideConsentButtonWhenBannerVisible'"
```

Comment:
Do we need the extra FF here? Looks like the button is already included within one.

Reply:
Oh, I think it was added by accidentally. I just deleted it.